currency.go

```diff
@@ -0,0 +1,37 @@
+package gofakeit
+
+import (
+	"math/rand"
+
+	"github.com/brianvoe/gofakeit/data"
+)
+
+// CurrencyInfo is a struct of currency information
+type CurrencyInfo struct {
+	Code string
+	Name string
+}
+
+// Currency will generate a struct with random currency information
+func Currency() *CurrencyInfo {
```

Comment:
Can we put the items of currency full in here and just get rid of CurrencyFull. Just a little redundent.